pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -100,6 +100,10 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, c *ConverterContext)
 				}
 			}
 		}
+
+		if iface.State == v1.InterfaceStateLinkDown {
```

Comment:
>  converter, net: Start domain with link state down
> 
> Unless the network interface is sriov, support starting VMs/VMIs with link state
> set to down.
> Note that since no state and 'up' state mean the same, there's no other value
> to address here. 'absent' state  is handled elsewhere.

I do not understand why this was needed. Please explain it.
By default we set nothing, why do we need to set anything?

Reply:
@EdDev has a point, it will be valuable if we could distinguish the three states in the VMI and the domain:
- empty - no link state (like the current state without this PR)
- down - link state down
- up - link state up